fvm/contractFunctionInvoker.go

```diff
@@ -46,13 +46,17 @@ func (i *ContractFunctionInvoker) Invoke(env Environment) (cadence.Value, error)
 	span.SetAttributes(i.logSpanAttrs...)
 	defer span.End()
 
+	runtimeEnv := env.BorrowCadenceRuntime()
```

Comment:
Should we just one per tx/script and reuse it here? I imagine we would need much less of them that way. 

Not sure if it really matters....

- It might save some time since you don't have to get it and return it to/from a channel each time
- It might save some memory since less `ReusableCadenceRuntimePool`s need to exist at a given time

Reply:
This is not an issue, but just a note:

during a transaction:
- you take one from the pool for the tx
- you take another one from the pool for fees (or any other operation)
- then you return both

This is not really an issue right now because the pool size is 1000 but if the pool size was 0 we would be creating and dropping 1 new runtime env each call.

Reply:
that's no worst than today.  so this doesn't seem like an issue